src/app/frontend/logs/logs_service.js

```diff
@@ -0,0 +1,44 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * Service class for logs colors management.
+ * @final
+ */
+class LogColorInversionService {
```

Comment:
I would like use this class for documentation. Because there is no export here I can not import it in controllers. But when I add export and I import this then lint complains because LogColorInversionService is not used (I want only this for documentation). How can I solve this issue?